holidays/locale/be/LC_MESSAGES/BY.po

```diff
@@ -48,8 +48,8 @@ msgid "Дзень Перамогі"
 msgstr ""
 
 #. Day of the National Coat of Arms of the Republic of Belarus, the National
-#. Flag of the Republic of
-#. Belarus and the National Anthem of the Republic of Belarus.
+#. Flag of the Republic
+#. of Belarus and the National Anthem of the Republic of Belarus.
```

Comment:
Belarus's `by.po` changes shouldn't be here:
- `holidays/locale/be/LC_MESSAGES/BY.po`
- `holidays/locale/en_US/LC_MESSAGES/BY.po`
- `holidays/locale/ru/LC_MESSAGES/BY.po`
- `holidays/locale/th/LC_MESSAGES/BY.po`

Reply:
These changes accidentally appeared in #2289, I just fixed it in `dev`.

Reply:
No, just merge actual `dev` in your branch.